### ISSUE TITLE
right to left

### ISSUE THREAD

**Issue by nickevan9** [Semper-Viventem/RoundedBackgroundSpan#1]
set background with text gravity right not working

**Comment by Semper-Viventem** [comment]
Yep. You are right. But I don't have time for implement gravity support :(

**Comment by mianaliasjad** [comment]
> set background with text gravity right not working

+1 if you can implement it.
